cloudferrylib/os/actions/check_sql.py

```diff
@@ -13,10 +13,22 @@
 # limitations under the License.
 
 from cloudferrylib.base.action import action
+from cloudferrylib.os.compute import nova_compute
+from cloudferrylib.os.image import glance_image
+from cloudferrylib.os.storage import cinder_database
+
+SQL = "SELECT TABLE_NAME FROM INFORMATION_SCHEMA.TABLES LIMIT 1"
 
 
 class CheckSQL(action.Action):
 
     def run(self, info=None, **kwargs):
-        self.cloud.mysql_connector.execute("SELECT TABLE_NAME FROM "
-                                           "INFORMATION_SCHEMA.TABLES LIMIT 1")
+        cfg = self.cloud.cloud_config
+        nova = nova_compute.NovaCompute(cfg, self.cloud)
```

Comment:
I guess we do not need to create new instances of these clients (compute, storage and image). They should be already initialized for this moment and we can find them in the `self.cloud.resources` dictionary.